cypress/integration/logistics/materialentnahme_complete_HU.js

```diff
@@ -122,6 +122,7 @@ describe('Create a purchase order and Material Receipts', function() {
 
   it('Visit referenced Material Receipt Candidates', function() {
     cy.openReferencedDocuments('M_ReceiptSchedule');
+    cy.waitForSaveIndicator();
```

Comment:
I don't think this particular wait is needed. ` cy.expectNumberOfRows` should do enough retries on its own.

Reply:
I have removed it for now but from time to time this test is failing but only on jenkins and that was a solution for the failure. Let's keep an eye on this in the future.